### PR TITLE
[FR] Update People

### PR DESCRIPTION
**Pull request by almaloste**
Update of some outdated People French Pages.

### CODE REVIEW

File: wiki/People/The_Team/Global_Moderation_Team/fr.md [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
-
 # Équipe de modération globale
 
 L **'équipe de modération globale** (*Global Moderation Team* ou *GMT* en anglais), dont les membres sont aussi connus sous le nom de **global mods** ou simplement **modos**, sont des membres du staff d'osu! dont l'objectif premier est d'encadrer la communauté sur le chat et le forum à l'aide du [règlement](/wiki/Rules).
 
-## Devenir un GM
+## Devenir un membre GMT
 
-Devenir un membre de l'équipe de modération globale est un processus lent et difficile. Les membres très actifs sont parfois désignés pour leur habilité à être bilingue, après une discussion avec le reste du staff pour leur possible promotion. Des GM sont généralement sélectionnés lorsqu'un canal de discussion a besoin d'un modérateur.
+Devenir un membre de l'équipe de modération globale est un processus lent et difficile. Les membres très actifs sont parfois désignés pour leur habilité à être bilingue, après une discussion avec le reste du staff pour leur possible promotion. Des GMT sont généralement sélectionnés lorsqu'un canal de discussion a besoin d'un modérateur.
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/Global_Moderation_Team/fr.md]:
```suggestion
Devenir un membre de l'équipe de modération globale est un processus lent et difficile. Les membres très actifs sont parfois désignés pour leur habilité à être bilingue après une discussion avec le reste du staff pour leur possible promotion. Des GMT sont généralement sélectionnés lorsqu'un canal de discussion a besoin d'un modérateur.
```

File: wiki/People/The_Team/fr.md [hunk 2]
```diff
@@ -1,60 +1,97 @@
...
-| [nuudles](https://osu.ppy.sh/users/21312) | Développeur d'osu!iPhone |
-| Phil | Conception du Forum |
-| [Sarumaru](https://osu.ppy.sh/users/9427)  ([Sarumaru@deviantART.com](http://sarumaru.deviantart.com/)) | Créateur et concepteur original de pippi |
-| [Sinistro](https://osu.ppy.sh/users/5530) | Meeting minutes, Importantes contributions à la FAQ |
-| [woc2006](https://osu.ppy.sh/users/1105845) | Développeur d'osu!, Développaur d'osu!mania |
-
-## User Groups
+| [Groupe de Nomination des Beatmaps](Beatmap_Nominators) | Contrôlent la qualité des Beatmaps certifiées "Qualified". |
+| [Développeurs](Developers) | Rendent le jeu incroyable en ajoutant de nouvelles fonctionnalités et en corrigeant les bugs. |
+| [Équipe de modération globale](Global_Moderation_Team) | Gardent un œil sur le Chat et le Forum. |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| [Équipe de modération globale](Global_Moderation_Team) | Gardent un œil sur le chat et le forum. |
```

File: wiki/People/The_Team/fr.md [hunk 3]
```diff
@@ -1,60 +1,97 @@
...
+| [Nomination Assessment Team](Nomination_Assessment_Team) | Managing the Beatmap Nominators |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| [Nomination Assessment Team](Nomination_Assessment_Team) | Gestion des nominations des beatmaps |
```

File: wiki/People/The_Team/fr.md [hunk 4]
```diff
@@ -1,60 +1,97 @@
...
+| [osu! alumni](https://osu.ppy.sh/groups/25) | Connus pour leurs nombreuses contributions. |
+| [Project Loved Team](Project_Loved_Team) | Recognising the beatmaps that the community loves most |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| [Project Loved Team](Project_Loved_Team) | Reconnaissent les beatmaps que la communauté aime le plus |
```

File: wiki/People/The_Team/fr.md [hunk 5]
```diff
@@ -1,60 +1,97 @@
...
-| [osu!news](https://osu.ppy.sh/groups/25) | Vous donne à lire du contenu de qualité depuis 2013. |
-| [Gérants des tournois](https://osu.ppy.sh/groups/26) | Responsible for handling officially supported osu! tournaments across all gamemodes. |
-| [Équipe de Certification de Qualité des Beatmaps](/wiki/People/Quality_Assurance_Team) | On vous aide à vous amuser sur osu!, en commencant par assurer la qualité des Beatmaps. |
+| ![][flag_US] [awp](https://osu.ppy.sh/users/2650) ([Barrel Roll Weekly](http://brw.twinkfish.com/)) | Aide et entretien général, Dessinateur de chibis |
+| ![][flag_NO] [cYsmix](https://osu.ppy.sh/users/272870) | Design audio et [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
+| ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) | Organisation et administration de la coupe du monde |
+| ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341) | Entretien général du site, Gestion de la communauté |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) ([blog](http://blog.echo.sh/)) | Développeur d'osu!, Intégration du Chat IRC dans le jeu, Entretien du site |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) ([blog](http://blog.echo.sh/)) | Développeur d'osu!, intégration du Chat IRC dans le jeu, entretien du site |

File: wiki/People/The_Team/fr.md [hunk 6]
```diff
@@ -1,60 +1,97 @@
...
+| ![][flag_US] [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour, diffusion de la Coupe du Monde et organisation de tournois |
+| ![][flag_NL] [Intermezzo](https://osu.ppy.sh/users/136842) | Développeur d'osu!, osz2/p2p backend |
+| ![][flag_US] Jim ([Brand New Games](http://www.bravegamer.com/)) | Concepteur original du site |
+| ![][flag_DE] [Loctav](https://osu.ppy.sh/users/71366) | Organisation et administration de la Coupe du monde, leadership du QAT et community management |
+| ![][flag_US] [LuigiHann](https://osu.ppy.sh/users/1079) ([LuigiHann@Deviantart](http://luigihann.deviantart.com/)) | Magnifique design général d'osu!, Contributions au skin par défaut et aux sets d'icônes |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| ![][flag_US] [LuigiHann](https://osu.ppy.sh/users/1079) ([LuigiHann@Deviantart](http://luigihann.deviantart.com/)) | Designer du magnifique design général d'osu!, contributions au skin par défaut et aux sets d'icônes |
```

File: wiki/People/The_Team/fr.md [hunk 7]
```diff
@@ -1,60 +1,97 @@
...
+| ![][flag_US] [mm201](https://osu.ppy.sh/users/30655) | Développeur d'osu!, Intégrateur des sliders |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| ![][flag_US] [mm201](https://osu.ppy.sh/users/30655) | Développeur d'osu!, intégrateur des sliders |
```

File: wiki/People/The_Team/fr.md [hunk 8]
```diff
@@ -1,60 +1,97 @@
...
+| ![][flag_US] [nuudles](https://osu.ppy.sh/users/21312) | Développeur d'osu!iPhone |
+| ![][flag_US] Phil | Conception du Forum |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| ![][flag_US] Phil | Conception du forum |
```

File: wiki/People/The_Team/fr.md [hunk 9]
```diff
@@ -1,60 +1,97 @@
...
+| ![][flag_US] [RBRat3](https://osu.ppy.sh/users/307202) | osu!painter, gars extraordinaire |
+| ![][flag_US] [Sarumaru](https://osu.ppy.sh/users/9427)  ([Sarumaru@deviantART.com](http://sarumaru.deviantart.com/)) | Créateur et concepteur original de pippi |
+| ![][flag_GR] [Sinistro](https://osu.ppy.sh/users/5530) | Meeting minutes, Importantes contributions à la FAQ |
+| ![][flag_DE] [Tom94](https://osu.ppy.sh/users/1857058) | Développement des pp |
+| ![][flag_CN] [woc2006](https://osu.ppy.sh/users/1105845) | osu! dev team, osu!mania mode dev |
+| ![][flag_JP][yelle](https://osu.ppy.sh/users/4916903) | Gestion du magasin osu! |
+| ![][flag_US] [ztrot](https://osu.ppy.sh/users/6347) | Professeur de osu!academy, Conception media/personnage |
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/fr.md]:
```suggestion
| ![][flag_US] [ztrot](https://osu.ppy.sh/users/6347) | Professeur de osu!academy, conception media/personnage |
```

File: wiki/People/The_Team/osu!_Alumni/fr.md [hunk 10]
```diff
@@ -4,17 +4,14 @@ outdated: true
 
 # Vétérans
 
-Les **vétérans**, connus sous le nom de **osu! Alumnis**, ont reçu leur distinction pour leurs énormes contributions à la communauté. La majorité d'entre eux sont inactifs désormais : il s'agit d'anciens membres du staff. Une rumeur court que lorsque peppy sera riche, il érigera une statue de chacun d'eux dans sa ville.
+Les **osu! Alumni** sont les personnes connues pour la contribution qu'ils ont eut avant de partir. are those known for their contributions who have since moved on. Si les ressources étaient disponibles, nous érigerions une statue pour chaque membre sur place publique.
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/osu!_Alumni/fr.md]:
```suggestion
Les **osu! Alumni** sont les personnes connues pour la contribution qu'ils ont eut avant de partir. Ils sont ceux connus pour leurs contributions qui ont depuis évolué. Si les ressources étaient disponibles, nous érigerions une statue pour chaque membre sur une place publique.
```

File: wiki/People/The_Team/osu!_Alumni/fr.md [hunk 11]
```diff
@@ -1,33 +1,31 @@
----
-outdated: true
----
+# osu!Alumni
 
-# Vétérans
-
-Les **osu! Alumni** sont les personnes connues pour la contribution qu'ils ont eut avant de partir. are those known for their contributions who have since moved on. Si les ressources étaient disponibles, nous érigerions une statue pour chaque membre sur place publique.
+Les **osu! Alumni** sont les personnes connues pour la contribution qu'ils ont eut avant de partir. Si les ressources étaient disponibles, nous érigerions une statue pour chaque membre sur place publique.
```
Comment (BlackJamesYT) [inline wiki/People/The_Team/osu!_Alumni/fr.md]:
```suggestion
Les **osu! Alumni** sont les personnes connues pour la contribution qu'ils ont eut avant de partir. Si les ressources étaient disponibles, nous érigerions une statue pour chaque membre sur la place publique.
```

File: wiki/People/fr.md [hunk 12]
```diff
@@ -1,8 +1,4 @@
----
-outdated: true
----
-
 # L'Équipe
```
Comment (cl8n) [inline wiki/People/fr.md]: this is supposed to be the title of "The Team", right?

File: wiki/People/The_Team/osu!_Alumni/fr.md [hunk 13]
```diff
@@ -1,36 +1,31 @@
----
-outdated: true
----
+# osu!Alumni
```
Comment (cl8n) [inline wiki/People/The_Team/osu!_Alumni/fr.md]:
```suggestion
# osu! Alumni
```

File: wiki/People/The_Team/osu!_Alumni/fr.md [hunk 14]
```diff
@@ -1,36 +1,31 @@
...
 
-Les **vétérans**, connus sous le nom de **osu! Alumnis**, ont reçu leur distinction pour leurs énormes contributions à la communauté. La majorité d'entre eux sont inactifs désormais : il s'agit d'anciens membres du staff. Une rumeur court que lorsque peppy sera riche, il érigera une statue de chacun d'eux dans sa ville.
+La [page de groupe des osu! Alumni](https://osu.ppy.sh/groups/16) liste de tous les membres.
 
-## Liste des osu! Alumnis actuels
-
-[Lien du groupe](https://osu.ppy.sh/groups/16)
-
-| Nom | Ancienne position |
+| Name | Past roles |
```
Comment (cl8n) [inline wiki/People/The_Team/osu!_Alumni/fr.md]: not translated?

File: wiki/People/The_Team/Global_Moderation_Team/fr.md [hunk 15]
```diff
@@ -1,25 +1,22 @@
...
 
 ## Rôle et responsabilités
 
 L'équipe de modération globale est responsable du maintien de la bonne ambiance sur le chat et le forum et sont prêts à donner des sanctions en cas d'infraction du règlement. Ils sont disponibles à tout joueur ayant des questions pertinentes concernant le jeu et son fonctionnement général ou diverses infractions.
 
-## Liste des GM
+## Liste des GMT
 
-*Veuillez noter que tous les GM peuvent également parler anglais en plus des langues listées ci-dessous.* - [Page du groupe](https://osu.ppy.sh/groups/4)
+*Veuillez noter que tous les GMT peuvent également parler anglais en plus des langues listées ci-dessous.* - [Page du groupe](https://osu.ppy.sh/groups/4)
```
Comment (cl8n) [inline wiki/People/The_Team/Global_Moderation_Team/fr.md]: updated in english

File: wiki/People/The_Team/osu!_Alumni/fr.md (hunk 14 above)
Comment (almaloste) [reply wiki/People/The_Team/osu!_Alumni/fr.md]: Maybe I forgot

File: wiki/People/fr.md (hunk 12 above)
Comment (almaloste) [reply wiki/People/fr.md]: Yes

**Comment by cl8n** [comment]
ive marked some of these outdated again cuz the people changed or small sections were added/changed

**Pull request merged**